docs/docs/integrations/llms/huggingface_endpoint.ipynb

```diff
@@ -20,7 +20,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from langchain_community.llms import HuggingFaceEndpoint"
+    "from langchain_community.llms.huggingface_endpoint import HuggingFaceEndpoint"
```

Comment:
Is this really needed? `HuggingFaceEndpoint` is an entry of `__all__` [[source](https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/llms/__init__.py#L899)], it should be importable with `from langchain_community.llms import HuggingFaceEndpoint`. 

![Screenshot-20240511-20-21-29](https://github.com/langchain-ai/langchain/assets/44113430/406efa25-4557-4c0b-afba-e5592e5290d8)

Reply:
Agree- I don't think it is needed.